cli/bin/cli.js

```diff
@@ -1,30 +1,39 @@
 #!/usr/bin/env node
 
 const open = require('open');
-const { showHelp, showVersion, showErrorMsg } = require('../index');
+const { color, showHelp, showVersion, showErrorMsg } = require('../index');
 
 async function main() {
   const validFlags = ['-v', '--version', '-h', ' --help'];
   const message = process.argv[2];
 
-  if (
-    message === undefined ||
-    message.toLowerCase() === '-h' ||
-    message.toLowerCase() === '--help'
-  ) {
-    showHelp();
-  } else if (
-    message.toLowerCase() === '-v' ||
-    message.toLowerCase() === '--version'
-  ) {
-    showVersion();
-  } else if (message.charAt(0) === '-' && !validFlags.includes(message)) {
-    showErrorMsg(message);
-  } else {
-    return await open(
-      `https://twitter.com/intent/tweet?text=${message}%0Acc%20%40sseraphini`,
+  if (message.length > 264) {
+    return console.log(
+      color('The maximum number of characters is 264!', 'red'),
```

Comment:
Currently we **don't have this limitation in the web version**, we let users write whatever they want, when they get to twitter they can "break the tweet into several"